agent-install/agent-install.sh

```diff
@@ -42,6 +42,8 @@ CSS_OBJ_PATH_DEFAULT='/api/v1/objects/IBM/agent_files'
 CSS_OBJ_AGENT_SOFTWARE_BASE='/api/v1/objects/IBM/agent_software_files'
 CSS_OBJ_AGENT_CERT_BASE='/api/v1/objects/IBM/agent_cert_files'
 CSS_OBJ_AGENT_CONFIG_BASE='/api/v1/objects/IBM/agent_config_files'
+MAX_HTTP_RETRY=5
+SLEEP_SECONDS_BETWEEN_RETRY=15
 
```

Comment:
I would suggest adding couple more changes to make this more robust.. You covered the cases where we download the packages but there are other curl usages that could fail... I suggest adding the following here

```
CURL_RETRY_PARMS="--retry 5 --retry-connrefused --retry-max-time 120"
```

and then in other cases that use curl that are not in your retry function, change
```
curl
```
to
```
curl ${CURL_RETRY_PARMS}
```

In the event that these could fail with some 50x errors, curl would retry